ide/projectuiapi/src/org/netbeans/spi/project/ui/support/NodeFactorySupport.java

```diff
@@ -174,7 +174,7 @@ public void run() {
                             NodeList<?> lst = factory.createNodes(project);
                             assert lst != null : "Factory " + factory.getClass() + " has broken the NodeFactory contract."; //NOI18N
                             lst.addNotify();
-                            List<?> objects = lst.keys();
+                            List objects = lst.keys();
```

Comment:
Is the switch to raw types here intented? This looks strange given the target of this PR.

Reply:
I've reverted the file NodeFactorySupport.java to the original.. It was too problematic to fix.. Will clean up later as there's plenty to do..